Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -12442,6 +12442,11 @@ - (void)_invokeAllActionsToPerformAfterPendingImageAnalysis:(WebKit::ProceedWith
 
 #endif // ENABLE(IMAGE_ANALYSIS)
 
+- (CALayer *)getTextIndicatorInstallationLayer
+{
+    return [self layer];
+}
+
 - (void)setUpTextIndicator:(Ref<WebCore::TextIndicator>)textIndicator
```

Comment:
Why is this (and the code below) still here? I would expect this to be one of the two consolidated pieces.